src/db/SQL.js

```diff
@@ -402,7 +402,7 @@ define([
 				return results;
 			}));
 			var store = this;
-			return new QueryResults(results, {
+			return QueryResults.default(results, {
```

Comment:
Oh right, TS does gripe about it. That is what led me to change it in the first place. "Only a void function can be called with the 'new' keyword".